README.md

```diff
@@ -3,11 +3,28 @@
 
 # Solrb
 
-Object-Oriented approach to Solr in Ruby. 
+Object-Oriented approach to Solr in Ruby.
 ## Installation
 
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'solr'
+gem 'solrb'
+```
+
+## Configuration
+
+```
+Solr.configure do |config|
+  config.url = http://yoursolrurl/
+
+  config.define_field_types do |t|
+    t.dynamic_field_type :text, solr_definition: '*_text'
+  end
+
+  config.define_fields do |f|
```

Comment:
I would call it slightly differently, it's essentially a mappings from field names to their types, so here I would call it 
```
config.map_types
or
config.define_field_types
or
config.configure_field_types
```